frontend/public/components/nav/items.tsx

```diff
@@ -84,6 +84,18 @@ export class ResourceClusterLink extends NavLink<ResourceClusterLinkProps> {
   }
 }
 
+export class ResourceChargebackLink extends NavLink<ResourceNSLinkProps> {
```

Comment:
We don't need a special link type. We should be able to use `HrefLink`

Reply:
ok, using `HrefLink`